## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 368092`

x86_64-linux running
x86_64-darwin running

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>rubyPackages_3_4.libv8</li>
    <li>rubyPackages_3_4.ovirt-engine-sdk</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>rubyPackages_3_4.uuid4r</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 385 packages built:</summary>
  <ul>
    <li>rubyPackages_3_4.actioncable</li>
    <li>rubyPackages_3_4.actionmailbox</li>
    <li>rubyPackages_3_4.actionmailer</li>
    <li>rubyPackages_3_4.actionpack</li>
    <li>rubyPackages_3_4.actiontext</li>
    <li>rubyPackages_3_4.actionview</li>
    <li>rubyPackages_3_4.activejob</li>
    <li>rubyPackages_3_4.activemodel</li>
    <li>rubyPackages_3_4.activerecord</li>
    <li>rubyPackages_3_4.activestorage</li>
    <li>rubyPackages_3_4.activesupport</li>
    <li>rubyPackages_3_4.addressable</li>
    <li>rubyPackages_3_4.ansi</li>
    <li>rubyPackages_3_4.ast</li>
    <li>rubyPackages_3_4.atk</li>
    <li>rubyPackages_3_4.awesome_print</li>
    <li>rubyPackages_3_4.backport</li>
    <li>rubyPackages_3_4.bacon</li>
    <li>rubyPackages_3_4.base64</li>
    <li>rubyPackages_3_4.benchmark</li>
    <li>rubyPackages_3_4.bigdecimal</li>
    <li>rubyPackages_3_4.bindata</li>
    <li>rubyPackages_3_4.builder</li>
    <li>rubyPackages_3_4.byebug</li>
    <li>rubyPackages_3_4.cairo</li>
    <li>rubyPackages_3_4.cairo-gobject</li>
    <li>rubyPackages_3_4.camping</li>
    <li>rubyPackages_3_4.certified</li>
    <li>rubyPackages_3_4.charlock_holmes</li>
    <li>rubyPackages_3_4.claide</li>
    <li>rubyPackages_3_4.clamp</li>
    <li>rubyPackages_3_4.cld3</li>
    <li>rubyPackages_3_4.cocoapods</li>
    <li>rubyPackages_3_4.cocoapods-acknowledgements</li>
    <li>rubyPackages_3_4.cocoapods-art</li>
    <li>rubyPackages_3_4.cocoapods-browser</li>
    <li>rubyPackages_3_4.cocoapods-clean</li>
    <li>rubyPackages_3_4.cocoapods-clean_build_phases_scripts</li>
    <li>rubyPackages_3_4.cocoapods-coverage</li>
    <li>rubyPackages_3_4.cocoapods-deintegrate</li>
    <li>rubyPackages_3_4.cocoapods-dependencies</li>
    <li>rubyPackages_3_4.cocoapods-deploy</li>
    <li>rubyPackages_3_4.cocoapods-disable-podfile-validations</li>
    <li>rubyPackages_3_4.cocoapods-downloader</li>
    <li>rubyPackages_3_4.cocoapods-expert-difficulty</li>
    <li>rubyPackages_3_4.standard-performance</li>
    <li>rubyPackages_3_4.stringio</li>
    <li>rubyPackages_3_4.syntax_tree</li>
    <li>rubyPackages_3_4.syntax_tree-haml</li>
    <li>rubyPackages_3_4.syntax_tree-rbs</li>
    <li>rubyPackages_3_4.taglib-ruby</li>
    <li>rubyPackages_3_4.temple</li>
    <li>rubyPackages_3_4.terminal-table</li>
    <li>rubyPackages_3_4.thor</li>
    <li>rubyPackages_3_4.thrift</li>
    <li>rubyPackages_3_4.tilt</li>
    <li>rubyPackages_3_4.timeout</li>
    <li>rubyPackages_3_4.tiny_tds</li>
    <li>rubyPackages_3_4.treetop</li>
    <li>rubyPackages_3_4.tty-color</li>
    <li>rubyPackages_3_4.tty-command</li>
    <li>rubyPackages_3_4.tty-option</li>
    <li>rubyPackages_3_4.typhoeus</li>
    <li>rubyPackages_3_4.tzinfo</li>
    <li>rubyPackages_3_4.uglifier</li>
    <li>rubyPackages_3_4.unf</li>
    <li>rubyPackages_3_4.unf_ext</li>
    <li>rubyPackages_3_4.unicode-blocks</li>
    <li>rubyPackages_3_4.unicode-display_width</li>
    <li>rubyPackages_3_4.unicode-name</li>
    <li>rubyPackages_3_4.unicode-scripts</li>
    <li>rubyPackages_3_4.unicode-types</li>
    <li>rubyPackages_3_4.uri</li>
    <li>rubyPackages_3_4.webmention</li>
    <li>rubyPackages_3_4.webrick</li>
    <li>rubyPackages_3_4.websocket-driver</li>
    <li>rubyPackages_3_4.websocket-extensions</li>
    <li>rubyPackages_3_4.whois</li>
    <li>rubyPackages_3_4.xcodeproj</li>
    <li>rubyPackages_3_4.xctasks</li>
    <li>rubyPackages_3_4.yard</li>
    <li>rubyPackages_3_4.zeitwerk</li>
    <li>rubyPackages_3_4.zookeeper</li>
    <li>ruby_3_4</li>
    <li>ruby_3_4.devdoc</li>
  </ul>
</details>

---

<details>

<summary>rubyPackages_3_4.uuid4r on aarch64-darwin</summary>

```
building '/nix/store/fkanlgxa5zwzk4kpbfx3w7w4x17dpf2h-ruby-3.4.1.drv'...
error: builder for '/nix/store/qa13gjs8jgfgqvlz50hm6dga9v82kw0v-libossp-uuid-1.6.2.drv' failed with exit code 2;
       last 25 log lines:
       > libtool: compile:  clang -I. -I. -DHAVE_CONFIG_H -O2 -pipe -c uuid_mac.c  -fno-common -DPIC -o .libs/uuid_mac.o
       > libtool: compile:  clang -I. -I. -DHAVE_CONFIG_H -O2 -pipe -c uuid_time.c  -fno-common -DPIC -o .libs/uuid_time.o
       > libtool: compile:  clang -I. -I. -DHAVE_CONFIG_H -O2 -pipe -c uuid_ui64.c  -fno-common -DPIC -o .libs/uuid_ui64.o
       > libtool: compile:  clang -I. -I. -DHAVE_CONFIG_H -O2 -pipe -c uuid_ui128.c  -fno-common -DPIC -o .libs/uuid_ui128.o
       > libtool: compile:  clang -I. -I. -DHAVE_CONFIG_H -O2 -pipe -c uuid_str.c  -fno-common -DPIC -o .libs/uuid_str.o
       > uuid_str.c:699:5: error: call to undeclared function '__VA_COPY_USE_yes'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       >   699 |     va_copy(ap_tmp, ap);
       >       |     ^
       > ./config.h:212:23: note: expanded from macro 'va_copy'
       >   212 | #define va_copy(d, s) __VA_COPY_USE(d, s)
       >       |                       ^
       > ./config.h:217:23: note: expanded from macro '__VA_COPY_USE'
       >   217 | #define __VA_COPY_USE __VA_COPY_USE_yes
       >       |                       ^
       > uuid_str.c:738:9: error: call to undeclared function '__VA_COPY_USE_yes'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       >   738 |         va_copy(ap_tmp, ap);
       >       |         ^
       > ./config.h:212:23: note: expanded from macro 'va_copy'
       >   212 | #define va_copy(d, s) __VA_COPY_USE(d, s)
       >       |                       ^
       > ./config.h:217:23: note: expanded from macro '__VA_COPY_USE'
       >   217 | #define __VA_COPY_USE __VA_COPY_USE_yes
       >       |                       ^
       > 2 errors generated.
       > make: *** [Makefile:109: uuid_str.lo] Error 1
       For full logs, run 'nix log /nix/store/qa13gjs8jgfgqvlz50hm6dga9v82kw0v-libossp-uuid-1.6.2.drv'.
error: 1 dependencies of derivation '/nix/store/w5lpnjh3v70kcyg1bq7jahrdixqd0i5g-ruby3.4-uuid4r-0.2.0.drv' failed to build
```

</details>

--- a/pkgs/development/interpreters/ruby/default.nix
+++ b/pkgs/development/interpreters/ruby/default.nix
@@ -441,8 +441,8 @@ in
   };
 
   ruby_3_4 = generic {
-    version = rubyVersion "3" "4" "0" "preview2";
-    hash = "sha256-RDzX7FSt5HhryXTOn11J8XKmD47chLWXt/4r0qlLg3E=";
+    version = rubyVersion "3" "4" "1" "";
+    hash = "sha256-PTheXSLTaLBkyBehPtjjzD9xp3BdftG654ATwzqnyH8=";
     cargoHash = "sha256-kdfNY8wVmSRR+cwEDYge/HDPRvdTNKLk/BhgqQeelOg=";
   };
 }

